code/script.js

```diff
@@ -6,4 +9,7 @@ function toggle() {
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById("section1").onclick = toggle;
+document.getElementById("section2").onclick = toggle;
+document.getElementById("section3").onclick = toggle;
+document.getElementById("section4").onclick = toggle;
```

Comment:
If you decide to go with semicolons, you should have them everywhere (so add them to the function as well). Apart from that, nice and clean!